src/scrot.c

```diff
@@ -942,6 +942,7 @@ static Imlib_Image scrotGrabShotMonitor(void)
     if (opt.monitor >= numScreens)
         errx(EXIT_FAILURE, "monitor %d not found", opt.monitor);
 
+    scrotAssert(screens); /* silence clang-tidy */
```

Comment:
It doesn't understand `XineramaQueryExtension` semantics. If `numScreens` is non zero, `screens` shouldn't be NULL.